docs/book/src/reference/rust_differences.md

```diff
@@ -34,3 +34,9 @@ struct MyStruct {
     field_two: bool,
 }
 ```
+
+## Memory Allocation
+
+In Rust, the borrow checker implements Rust's [ownership system](https://doc.rust-lang.org/1.8.0/book/ownership.html)
+
+In Sway, there is no borrow checker.  This means there is no concept of ownership, borrowing, or lifetimes.  Instead, objects are copied and moved similar to C++.  Also Sway does not have any destructors nor `Drop` traits.  This means allocated memory lives for the entire transaction and is not deallocated until the end of the transaction.  A transaction can contain 65 MB of memory.
```

Comment:
Do we have a source for the 65 MB number and can that be linked to the spec documentation if that's where it's from? Since that may change in the future.

Reply:
cc @FuelLabs/client

Reply:
I couldn't find it in the specs, but I linked to where it is in our code